### PR TITLE
Add meta tags to improve website's SEO

### PR DESCRIPTION
**Pull request by psantos10**
More tags can be added in future. these are the basics. issue #75

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
-  <!-- Enable responsiveness on mobile devices-->
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
 
+  <link href="http://gmpg.org/xfn/11" rel="profile">
+
+  <meta name="description" content="Lessons about the Elixir programming language, inspired by Twitter’s Scala School.">
+  <meta name="author" content="Sean Callan">
+
+  <meta property="og:title" content="Elixir School - Lessons about the Elixir programming language">
+  <meta property="og:image" content="http://elixir-lang.org/images/logo/logo.png">
```
Comment (pragmaticivan) [inline _includes/head.html]: We need a proper image following Facebook post size standards for this one.

File: _includes/head.html [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+  <meta property="og:description" content="Lessons about the Elixir programming language, inspired by Twitter’s Scala School.">
+
+  <meta name="google-site-verification" content="6ZHfETSmHjwNySjadWFTG-NUsVD_Lm2T46jdmYy43Zg" />
```
Comment (pragmaticivan) [inline _includes/head.html]: Where is this code coming from? does @doomspork has management access to it?

File: _includes/head.html (hunk 1 above)
Comment (pragmaticivan) [inline _includes/head.html]: Here's the link for more info: https://developers.facebook.com/docs/sharing/best-practices#images

File: _includes/head.html (hunk 2 above)
Comment (psantos10) [inline _includes/head.html]: No @doomspork has no access. But after setup I can pass the access. I need a way to have a basic report as I will improve the SEO. It come from my own Google Account.

Comment (doomspork) [inline _includes/head.html]: @psantos10 I've already verified the domain through DNS, what is this for?  I can give you access to the GA accounts for ElixirSchool.

Comment (psantos10) [inline _includes/head.html]: @doomspork  is for have access to the Google Webmaster Tool https://www.google.com/webmasters/tools/home?hl=en

Comment (pragmaticivan) [inline _includes/head.html]: @psantos10 you have a couple of extra options for that: print

Comment (psantos10) [inline _includes/head.html]: @pragmaticivan  I am using HTML tag. Which another better option can you recommend for this purpose?

Comment (pragmaticivan) [inline _includes/head.html]: If @doomspork grant you google Analytics access you can use your account so both of you guys will have access to the same validation.

Comment (doomspork) [inline _includes/head.html]: @psantos10 I already setup Elixir School in the Webmaster Tools, using the Domain name provider method.  I'd be glad to add you as a user if you could share your email address.

Comment (doomspork) [inline _includes/head.html]: Alright, @psantos10 has been given access.  This line can be removed.

File: _includes/head.html [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
 
+  <link href="http://gmpg.org/xfn/11" rel="profile">
+
+  <meta name="description" content="Lessons about the Elixir programming language, inspired by Twitter’s Scala School.">
+  <meta name="author" content="Sean Callan">
+
+  <meta property="og:url" content="https://elixirschool.com/">
+  <meta property="og:site_name" content="ElixirSchool">
+  <meta property="og:title" content="Elixir School - Lessons about the Elixir programming language">
```
Comment (pragmaticivan) [inline _includes/head.html]: What about adding the title logic here as well?

Comment (psantos10) [inline _includes/head.html]: @pragmaticivan I am trying to figure out a way to reuse the logic

Comment (pragmaticivan) [inline _includes/head.html]: `<meta property="og:title" content="{% if page.title == "Home" %} {{ site.title }} &middot; {{ site.tagline }}  {% else %} {{ page.title }} &middot; {{ site.title }}  {% endif %}">` ?

Comment (psantos10) [inline _includes/head.html]: what about this one:

```
  <meta property="og:title" content="{% include title.html %}">
  <meta property="og:description" content="Lessons about the Elixir programming language, inspired by Twitter’s Scala School.">
  <meta property="og:image" content="https://elixirschool.com/assets/fb_share.jpg">

  <title>
    {% include title.html %}
  </title>
```

**Comment by pragmaticivan** [comment]
I think it's fine for me this title change, but let's wait @doomspork .

**Comment by doomspork** [comment]
@psantos10 I have no objections to this change, have you verified that it works or should I pull the changes?

**Comment by psantos10** [comment]
@doomspork  It is working. I tested it locally

**Comment by doomspork** [comment]
Great!  Thank you @psantos10 :+1:

**Pull request merged**